mypy/fastparse.py

```diff
@@ -804,7 +805,8 @@ def visit_raw_str(self, s: str) -> Type:
         return parse_type_comment(s.strip(), line=self.line)
 
     def generic_visit(self, node: ast35.AST) -> None:
-        raise TypeCommentParseError(TYPE_COMMENT_AST_ERROR, self.line)
+        raise TypeCommentParseError(TYPE_COMMENT_AST_ERROR, self.line,
```

Comment:
unfortunately I think we still need the getattr here... not all AST nodes come with column info, only the ones deriving from stmt, expr, etc...

(https://github.com/python/typeshed/blob/master/third_party/3/typed_ast/ast35.pyi)